src/components/app/Home.js

```diff
@@ -426,18 +426,14 @@ class Home extends React.PureComponent<HomeProps, HomeState> {
 
   _renderShortcuts() {
     return (
-      <p>
-        <div>
```

Comment:
It's forbidden to have a <div> in a <p>. React was outputting a warning before (also in the browser's console) but it was swallowed by our test runner previously. Now with this library React warnings are much more visible so I wanted to fix it here. 

But I'd be happy to extract it to another PR if you prefer.

I checked the rendering is exactly the same, but I may have missed something.